howso/prediction_stats.amlg

```diff
@@ -60,6 +60,7 @@
 	;		If null, will be set to k. default is null
 	; num_robust_influence_samples_per_case: optional, Specifies the number of robust samples to use for each case for robust contribution computations.
 	;				  Defaults to 300 + 2 * (number of features).
+	; prediction_stats_features: list of string, optional. List of features to use when calculating prediction stats. Should contain all action and context features desired.
```

Comment:
I feel like I'd rather this parameter just be "features" or something like that. This feels slightly redundant and verbose